pkg/tsdb/cloudwatch/log_query_test.go

```diff
@@ -205,20 +212,23 @@ func TestGroupKeyGeneration(t *testing.T) {
 		aws.String("fakelog-a"),
 		aws.String("fakelog-b"),
 		aws.String("fakelog-c"),
+		nil,
 	})
 
 	streamField := data.NewField("stream", data.Labels{}, []*string{
 		aws.String("stream-a"),
 		aws.String("stream-b"),
 		aws.String("stream-c"),
+		aws.String("stream-d"),
 	})
 
 	fakeFields := []*data.Field{logField, streamField}
-	expectedKeys := []string{"fakelog-astream-a", "fakelog-bstream-b", "fakelog-cstream-c"}
+	expectedKeys := []string{"fakelog-astream-a", "fakelog-bstream-b", "fakelog-cstream-c", "stream-d"}
 
 	assert.Equal(t, expectedKeys[0], generateGroupKey(fakeFields, 0))
 	assert.Equal(t, expectedKeys[1], generateGroupKey(fakeFields, 1))
 	assert.Equal(t, expectedKeys[2], generateGroupKey(fakeFields, 2))
+	assert.Equal(t, expectedKeys[4], generateGroupKey(fakeFields, 3))
```

Comment:
Shouldn't the index be 3, the length of expectedKeys is only 4?
```suggestion
	assert.Equal(t, expectedKeys[3], generateGroupKey(fakeFields, 3))
```